### DISCUSSION TITLE
Cannot download Youtube video using MS Edge Browser's VDH Version 9.1.0.43 (stable)

### DISCUSSION THREAD

**Discussion by rickygai** [aclap-dev/video-downloadhelper#1329] [Help and Q&A]
### Your question

Microsoft Windows 11 Version 22H2 (OS Build 22621.4249)
Microsoft Edge Version 129.0.2792.65 (Official build) (64-bit)
Video DownloadHelper Version 9.1.0.43 (stable), 

I cannot download a Youtube video link below.

[Drawing Graphics In Your Apps with Maui.Graphics](https://www.youtube.com/watch?v=SH8oQZp0oF0)

Please advise.

### Addon & CoApp version and details

[AdBlocker Ultimate](https://microsoftedge.microsoft.com/addons/detail/adblocker-ultimate/pciakllldcajllepkbbihkmfkikheffb)

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 24H2 build 26100.1882, Firefox 131.0 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.  You didn't mention your CoApp version, but I'm assuming it's 2.0.19.  Since you mentioned it, I'm running uBlock Origin.

For me, that video downloaded without incident in VDH & played without incident in VLC.  I didn't actually sit & watch it, just skimmed it, but it had video & audio at each of my sample points right to the end.

Of the differences between our setups, I suspect the issue is the different version of VDH due to your being on Edge while I, being on Firefox, have access to a more recent version, being the beta.  I suppose my advice is to either switch to Firefox & get the VDH beta, or wait for an updated VDH for Edge.  Well, to be complete, I suppose I should add a couple of further possibilities: give up, use another tool.

**Reply by rickygai** [reply]
Companion App installed (v2.0.19)
